setup.py

```diff
@@ -14,7 +14,7 @@
         'pytest',
         'pytest-cov',
     ],
-    ':python_version < "3.4"': [
+    'python_version < "3.4"': [
```

Comment:
Updated to 18.7, same exception. 
![it is a mystery](http://itisamystery.com/iiam.gif)